agent/ebs/watcher.go

```diff
@@ -126,7 +126,9 @@ func (w *EBSWatcher) HandleEBSResourceAttachment(ebs *apiebs.ResourceAttachment)
 	}
 
 	// start EBS CSI Driver Managed Daemon
-	if runningCsiTask := w.taskEngine.GetDaemonTask(md.EbsCsiDriver); runningCsiTask != nil {
+	// We want to avoid creating a new CSI driver task if there's already one that's not been stopped.
+	// TODO: Include unit tests
+	if runningCsiTask := w.taskEngine.GetDaemonTask(md.EbsCsiDriver); runningCsiTask != nil && !runningCsiTask.GetKnownStatus().Terminal() {
```

Comment:
this ties into the check above in `IsManagedDaemonTask`?  can we use the Terminal Check above also?
```
if !task.IsInternal || taskStatus >= apitaskstatus.TaskStopped {
```

Reply:
sure